.buildkite/pipeline.yml

```diff
@@ -24,6 +24,13 @@ steps:
       cpu: "8"
       memory: "4G"
 
+  - label: ":go: :windows: Run unit tests"
+    key: unit-tests-windows
+    command: ".buildkite/scripts/unit_tests_windows.ps1"
+    agents:
+      provider: "gcp"
+      image: "family/ci-windows-2022"
```

Comment:
Great to have this running also in Windows :+1: 

I would also add this step as a dependency of the step related to integration tests:

```yaml
  - label: ":pipeline: Trigger Integration tests"
    command: ".buildkite/pipeline.trigger.integration.tests.sh | buildkite-agent pipeline upload"
    depends_on:
      - step: check-static
        allow_failure: false
      - step: unit-tests-linux
        allow_failure: false
      - step: unit-tests-windows
        allow_failure: false
```
Doing so, if the unit tests in windows fail, integration tests won't be triggered.

Reply:
Added, thanks :+1: